lib/plugins/aws/invoke.js

```diff
@@ -70,6 +78,13 @@ class AwsInvoke {
       Payload: Buffer.from(JSON.stringify(this.options.data || {})),
     };
 
+    if (this.options.context) {
+      if (typeof this.options.context === 'object') {
```

Comment:
Let's do reverse condition `if (typeof this.options.context !== 'string')`, it'll be more bulletproff